tests/RepositoryForms/FieldType/Mapper/UserAccountFieldValueFormMapperTest.php

```diff
@@ -19,6 +20,15 @@ protected function setUp()
         parent::setUp();
 
         $formRoot = $this->getMockBuilder(FormInterface::class)->getMock();
+        $formRoot->expects($this->once())
+            ->method('getData')
+            ->willReturnCallback(function () {
+                $data = new \stdClass();
+                $data->contentType = $this->createMock(ContentType::class);
+
+                return $data;
```

Comment:
Why are you not returning proper object?

Reply:
you can just use `willReturn()` method